presto-main/src/main/java/com/facebook/presto/operator/scalar/ReflectionParametricScalar.java

```diff
@@ -303,6 +295,15 @@ public ImplementationParser(String functionName, Method method, Map<Set<TypePara
                 literalParameters.addAll(asList(literalParametersAnnotation.value()));
             }
 
+            SqlType returnType = method.getAnnotation(SqlType.class);
```

Comment:
We need to parse LiteralParameters first in order to parse returnType properly now.